docs/static/running-logstash-kubernetes.asciidoc

```diff
@@ -0,0 +1,6 @@
+[[running-logstash-kubernetes]]
+=== Running Logstash on Kubernetes
+Starting from Logstash 8.4, Logstash can run on Kubernetes cluster through Elastic Cloud on Kubernetes (ECK) in technical preview.
```

Comment:
This should be at least [8.6.0](https://github.com/elastic/cloud-on-k8s/blob/main/pkg/controller/common/version/version.go#L36), and maybe even [8.7.0](https://github.com/elastic/logstash/pull/14949) for full support.

Reply:
The version number is optional in Logstash doc I believe. 
ECK side mentioned "Running Logstash on ECK is compatible only with Logstash 8.7+" should be enough.
How about

```suggestion
{ls} runs on a Kubernetes cluster through Elastic Cloud on Kubernetes (ECK) in technical preview.
```

Reply:
I'm good with that